### DISCUSSION TITLE
Status: Checker HTTP error - 500

### DISCUSSION THREAD

**Discussion by RonnanWerneck** [shirokumacode/overwatch-omnic-rewards#20] [Q&A]
This error started yesterday, and it has been like this since. I pressed "check now" a few times but the error is still showing up. Any idea why?
![imagem_2022-05-31_133331901](https://user-images.githubusercontent.com/67077354/171226803-6aea3626-cae6-4723-9225-10eab7c32667.png)

**Comment by shirokumacode** [comment]
Check issue #21 for more info. This should be fixed for now.

Also yesterday was a different error (page incorrectly formatted), because the page was not properly formatted when they added the new banner at the top. I guess it wasn't intended and Blizzard fixed it. Checker HTML error 500 is because overwatchleague.com is returning HTML error code 500 in the response, although I they still send the normal page and not an error page. Which is kinda dumb, not normal/standard and why it seems fine even if you check the page with your browser.

**Reply by RonnanWerneck** [reply]
Thanks for the support! I forgot the name of the error so that's why I didn't type it here but yeah, the "page incorrectly formmated" was showing up yesterday for me. Thanks for clarifying :)
